### PR TITLE
Training Videos page with refactored modal

### PR DESCRIPTION
**Pull request by andrewjcasal**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/_includes/js/store.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 (function() {
+  window.LANDING_UTILS = window.LANDING_UTILS || {};
```
Comment (sinan) [inline docs/_includes/js/store.js]: this line is not necessary.

File: docs/_includes/js/training.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+(function() {
+
+  window.LANDING_UTILS = window.LANDING_UTILS || {};
```
Comment (sinan) [inline docs/_includes/js/training.js]: this line is not necessary.

File: docs/_includes/js/trial.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+(function() {
+
+  window.LANDING_UTILS = window.LANDING_UTILS || {};
```
Comment (sinan) [inline docs/_includes/js/trial.js]: this line is not necessary.

**Review by sinan — CHANGES_REQUESTED** [review]
please share some screenshots, or even better a screencast.

**Comment by andrewjcasal** [comment]
http://screencast.com/t/4Mg6H4HBzU19

**Review by sinan — APPROVED** [review]

**Pull request merged**